File: Medium-Questions/02-Star-Number-(Fa)/C#/Two.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Two
{
    class Program
    {
        static void Main(string[] args)
        {
            // get input from user
            Console.Write("Enter a Number: ");
            string user = Console.ReadLine();
            
            // convert int to string
            int res = Int32.Parse(user);

            // this for loop is for row
            for (int i = 1; i <= res; i++)
            {
                // this for loop is for print Star
                for (int j = 0; j < i; j++)
                {
                    Console.Write("*");
                }
                // print New Line
                Console.WriteLine("");
            }

            Console.ReadKey();
        }

    }
}

File: Medium-Questions/02-Star-Number-(Fa)/C/Two.c
#include <stdio.h>


int main(void)
{
    // Get input from user
    int user = 0;
    printf("Enter a Number: ");
    scanf("%i", &user);

    if (user < 1)
    {
        printf("Invalid input :(")
        return 1;
    }

    // this for loop is for Row
    for (int i = 1; i <= user ;i++)
    {
        for (int j = 0; j < i ; j++)
        {
            printf("*");
        }
        // Print New Line in end of each Row
        printf("\n");

    }
    return 0;
}